pkg/prometheus/testdata/ScrapeConfigSpecConfig_IonosSD_withBasicAuth.golden

```diff
@@ -0,0 +1,28 @@
+global:
+  evaluation_interval: 30s
+  scrape_interval: 30s
+  external_labels:
+    prometheus: default/test
+    prometheus_replica: $(POD_NAME)
+scrape_configs:
+- job_name: scrapeConfig/default/testscrapeconfig1
+  ionos_sd_configs:
+  - basic_auth:
+      username: ""
+      password: ""
+    proxy_url: http://no-proxy.com
+    no_proxy: 0.0.0.0
+    proxy_from_environment: true
+    proxy_connect_header:
+      header:
+      - value
+    enable_http2: true
+    datacenter_id: 11111111-1111-1111-1111-111111111111
+    follow_redirects: true
+    enable_http2: true
```

Comment:
enable_http2  duplicate

Reply:
This probably happened because of the duplication above.

>> it's already called above (at L4102)?